rs/embedders/src/wasmtime_embedder.rs

```diff
@@ -1030,6 +1037,8 @@ impl WasmtimeInstance {
         self.instance_stats.wasm_mmap_count += access_results.wasm_mmap_count;
         self.instance_stats.wasm_mprotect_count += access_results.wasm_mprotect_count;
         self.instance_stats.wasm_copy_page_count += access_results.wasm_copy_page_count;
+        self.instance_stats.wasm_sigsegv_handler_duration +=
+            access_results.wasm_sigsegv_handler_duration;
```

Comment:
Is accumulation the right thing to do for durations? I suppose everything else is a count/number of pages, so it's more clear to me that we should accumulate.

Reply:
IMO these should all be simple assignments. I can fix them all if you agree...